content/01_WhyRx.md

```diff
@@ -1,241 +1,126 @@
+
+Rx builds on LINQ (Language Integrated Query). This enables us to use the query syntax shown above (or you can use the explicit function call approach that some .NET developers prefer). LINQ is widely used in .NET both for data access (e.g., in Entity Framework Core), but also for working with in-memory collections (with LINQ to Objects), meaning that experienced .NET developers will tend to feel at home with Rx. Crucially, LINQ is a highly composable design: you can connect operators together in any combination you like, expressing potentially complex processing in a straightforward way.
+
+
+## When is Rx appropriate?
+
+Rx is designed for processing sequences of events, meaning that it suits some scenarios better than others. The next sections describe some of these scenarios, and also cases in which it is a less obvious match but still worth considering. Finally, we describe some cases in which it is possible to use Rx but where alternatives are likely to be better.
+
+
+### Good Fit with Rx
+
+Rx is well suited to representing events that originate from outside of your code, and which your application needs to respond to, such as:
+
+- Integration events like a broadcast from a message bus or a push event from WebSockets API or other low latency middleware like [Azure Event Grid](https://azure.microsoft.com/en-gb/products/event-grid/), [Azure Event Hubs](https://azure.microsoft.com/en-gb/products/event-hubs/) and [Azure Service Bus](https://azure.microsoft.com/en-gb/products/service-bus/)
+- Reports from monitoring devices such as a flow sensor in a water utility's infrastructure, or the monitoring and diagnostic features in networking equipment
+- Location data from mobile systems such as [AIS](https://github.com/ais-dotnet/) messages from ships, or automotive telemetry
+- Operating system events such as filesystem activity, or WMI events
+- Road traffic information, such as notifications of accidents or changes in average speed
+- Integration with a [Complex Event Processing (CEP)](https://en.wikipedia.org/wiki/Complex_event_processing) engine
+- UI events such as mouse movement or button clicks
+
+Rx is also good way to model domain events—these may occur as a result of some of the events just described, but after processing them to produce events that more directly represent application concepts. These might include:
+
+- Property or state changes on domain objects such as "Order Status Updated", or "Registration Accepted"
+- Changes to collections of domain objects, such as "New Registration Created"
+
+Events might also represent insights derived from incoming events and historical data such as:
+
+- A broadband customer might have become an unwitting participant in a DDoS attack
+- [CNC](https://en.wikipedia.org/wiki/Numerical_control) [Milling Machine](https://en.wikipedia.org/wiki/Milling_(machining)) MFZH12's number 4 axis bearing is exhibiting signs of wear at a significantly higher rate than the nominal profile
```

Comment:
You could use an AIS example - i.e. two vessels exhibiting suspicious behaviour, as a way to tie back to the previous list too.